npm-shrinkwrap.json

```diff
@@ -1,145 +1,145 @@
 {
```

Comment:
There appear to be _a lot_ of changes in this file, even though production `dependencies` aren't modified in the package.json. Is this intentional?

Reply:
I just run the npm shrinkwrap. Shows that the file was outdated then the npm just fix it. If you check the pckg.json file just the dev deps were updated.

Reply:
If you're not changing _production_ dependencies (only _dev_ dependencies), then you don't need to run `npm shrinkwrap`. If you still _want_ to run `npm shrinkwrap` and the dependency set changes like this, then you may need to consider if additional testing/verification needs to be done -- where the initial aim of your pull request was to change something in build/dev, changing the shrinkwrap file now results in a different program (for reasons unrelated to your build/dev time changes).